scikitplot/plotters.py

```diff
@@ -92,6 +146,26 @@ def plot_confusion_matrix(y_true, y_pred, labels=None, title=None, normalize=Fal
         cm = np.around(cm, decimals=2)
         cm[np.isnan(cm)] = 0.0
 
+    if true_labels is None:
+        true_classes = classes
+    else:
+        validate_labels(classes, true_labels, "true_labels")
+
+        true_label_indexes = np.where(np.in1d(classes, true_labels))
+
+        true_classes = classes[true_label_indexes]
+        cm = cm[true_label_indexes]
+
+    if pred_labels is None:
+        pred_classes = classes
+    else:
+        validate_labels(classes, pred_labels, "pred_labels")
+
+        pred_label_indexes = np.where(np.in1d(classes, pred_labels))
```

Comment:
Same here, 

`pred_label_indexes = np.where(np.in1d(classes, pred_labels))`

Although for this one, you'll want to change L167 to
`cm = cm[:, pred_label_indexes]`